keyboards/matchstickworks/southpad/rev2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2023 Logan Butler*/
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT(
+		KC_BSPC, KC_EQL,  LSFT(KC_9),LSFT(KC_0),
+        KC_PMNS, KC_PAST, KC_PSLS, TG(1),
+        KC_PPLS, KC_P9,   KC_P8,   KC_P7,
+        KC_P6,   KC_P5,   KC_P4,
+        KC_PENT, KC_P3,   KC_P2,   KC_P1,
+        KC_PDOT, KC_P0
+	),
```

Comment:
Ideally, this should be rearranged to more closely match the normal numpad layout. 
```suggestion
	[0] = LAYOUT(
		KC_BSPC, KC_EQL,  LSFT(KC_9),LSFT(KC_0),
        KC_PMNS, KC_PAST, KC_PSLS, TG(1),
                 KC_P9,   KC_P8,   KC_P7,
        KC_PPLS, KC_P6,   KC_P5,   KC_P4,
                 KC_P3,   KC_P2,   KC_P1,
        KC_PENT, KC_PDOT,      KC_P0
	),
```

eg: 
https://github.com/qmk/qmk_firmware/blob/master/layouts/default/numpad_6x4/default_numpad_6x4/keymap.c

While I know it's mirrored, consistency would be ideal.

Reply:
I've adjusted the formatting to tab lines 12 and 14 over, but the pcd as is wired doesn't allow moving KC_PENT and KC_PPLS down to those lines. The formatting will be implemented in next commit.